Resources/ServerInfo/Rules.txt

```diff
@@ -13,70 +13,67 @@
-- Do not use text speech like "omg", or "XD".
-- Referring to gamey terms like "admins", or "RNG" is a no go.
-- Do not use emotes to bypass chat filters and generally try to be at least medium effort with them, don't do things like, "John Doe motions with his hands to signal syndicate agents west of you."
-- Don't respond to events in a highly unbelievable manner, try to stick to how your character would react.
-
-[color=#7ade00]10.[/color] Do not disconnect, ghost, suicide, or otherwise intentionally permanently remove yourself from the round as important roles or while facing in-character punishment such as jail time without ahelping before hand. You don't need a response back, but failure to do so will result in consequences.
+[color=#7ade00]10.[/color] Do not disconnect, ghost, suicide, or otherwise intentionally permanently remove yourself from the round as an important role or while facing in-character punishment such as jail time without ahelping beforehand. You do not need to wait for a response, however, failure to notify admins will result in consequences.
 - This does not apply to cryo.
 
 
 [color=#7ade00]11.[/color] Your character name should be believable and realistic.
-- Your character name should not mimic popular real life or fictional people. Even if it's "clever" plays on them, like "White Walter."
-- Your character's name can't be phonetic word plays, like, "Mike Oxlong" or "Ben Dover."
-- Your name shouldn't be overly dumb or gamey either, like, "John Graytide."
-- Names should not include titles, such as, "Doctor", "Sergeant", or "Father."
+- Your character name should not mimic popular real-life or fictional people, even if it is a "clever" plays on them, such as "White Walter."
+- Your character's name should not be phonetic word plays, such as "Mike Oxlong" or "Ben Dover."
+- Your name should not be overly dumb or gamey either, such as "John Graytide."
+- Names should not include titles, such as "Doctor," "Sergeant," or "Father."
 
-[color=#7ade00]12.[/color] Do not prepare items or locations for "just in case" purposes. Referred to as powergaming. Standard Operating Procedure dictates how areas should be locked down and what gear should be held during any given circumstance. Minor things like getting a crowbar on round start is fine, seeking out weapons is not.
+[color=#7ade00]12.[/color] Do not prepare items or locations for "just in case" purposes (also referred to as "powergaming"). Standard Operating Procedure (SOP) dictates how areas should be locked down and what gear should be held and/or equipped during any given circumstance. Minor things such as getting a crowbar on round start is fine, seeking out weapons is not.
```

Comment:
I would not say that preparing items ahead of time is the only kind of power gaming. Maybe instead of "also referred to as 'powergaming'" say something along the lines of "this is commonly considered powergaming"?

Reply:
If I'm remembering correctly, this *is* what upstream defines it as (also, yes, preparing items that may give you an advantage over other players without proper reason is by all means powergaming)

Reply:
Fair enough, if that is what upstream does it is fine to keep it. My point was there are other forms of powergaming than just prepping items in advance (eg. chugging chems). Prepping items in advance is obviously also powergaming.